File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        reverse_number = 0
        original_number = x
        while (x>0):
            remainder = x%10
            reverse_number = reverse_number * 10 + remainder
            x //= 10
        if original_number == reverse_number:
            return True
        return False

File: 0009-palindrome-number/0009-palindrome-number.ts
function isPalindrome(x: number): boolean {
    const original_number: number = x
    let reverse_number: number = 0
    while (x > 0){
        let last_digit: number = x % 10
        x = Math.floor(x / 10)
        reverse_number = reverse_number * 10 + last_digit
    }
    return original_number === reverse_number ? true : false
};